addons/point_of_sale/static/src/app/models/related_models.js

```diff
@@ -593,19 +593,19 @@ export function createRelatedModels(modelDefs, modelClasses = {}, opts = {}) {
                         }
                     } else if (type === "set") {
                         const linkedRecs = record[name];
-                        const existingRecords = items.filter((record) =>
-                            exists(comodelName, record.id)
-                        );
+                        const newRecords = command[1]
+                            .map((id) => models[comodelName].get(id))
+                            .filter(Boolean);
```

Comment:
you don't need to make this change. Just make sure that when you call `["set", ...items]`, the `items` exist.

Reply:
Sometime the record didn't exist in the related models, so I prefer to not send it for processing